### ISSUE TITLE
DAG as a way to visualise decisions of the trained BART tree

### ISSUE THREAD

**Issue by sunPi** [kapelner/bartMachine#51]
First of all, thank you for this implementation it works really well in R and is easy to use as well as efficient.

There are a lot of implemented functions that allow the user to measure the performance of a BART model. Other metrics are normally implemented by the user (e.g. correlation or R2 etc.). What about explaining the model? I don't know if there is an implemented way to draw the final decision tree that was made by the Bart machine. 

For example, I am looking for a way to generate a DAG graph from the decisions made by the final BART tree. This way we could calculate the importance/contribution of each feature to the final model performance, possibly implementing SHAP values as well. 

In short, is there a way to draw the BART tree as a DAG or any other way to measure feature contribution?

**Comment by kapelner** [comment]
Bart is a sum of trees model. If you want one tree, use the rpart package
which also has nice visuals. If you want to "explain" the Bart model, good
luck. Nobody knows how to explain machine learning. Check out Stephen
Wolfram's testimony before the United States congress on YouTube.
